go/executor/executor.go

```diff
@@ -48,6 +48,7 @@ const (
 	sqlflowToRunContextKeySelect = "SQLFLOW_TO_RUN_SELECT"
 	sqlflowToRunContextKeyInto   = "SQLFLOW_TO_RUN_INTO"
 	sqlflowToRunContextKeyImage  = "SQLFLOW_TO_RUN_IMAGE"
+	sqlflowToRunProgramFolder    = "/opt/sqlflow/run"
```

Comment:
Why this folder? Is it a default folder when building a docker image or starting a docker container?

Reply:
Yes, it's the default folder where user put runnable at. We need specify this convention in our user customized runnable document.